DEVBOX_VERSION

```diff
@@ -0,0 +1 @@
+1
```

Comment:
Is this integer increment? Can you do semantic versioning?

Reply:
It's just any string. We can do semver.

Reply:
ooo could we do that?